sys/net/network_layer/fib/fib.c

```diff
@@ -30,6 +30,24 @@
 #include "ng_fib.h"
 #include "ng_fib/ng_fib_table.h"
 
+#ifdef MODULE_NG_IPV6
+#include "net/ng_ipv6.h"
+static char addr_str[NG_IPV6_ADDR_MAX_STR_LEN];
+#endif
+
+#ifdef MODULE_NG_IPV6
+    #define NG_FIB_ADDR_PRINT_LEN       39
```

Comment:
it is necessary to omit parens here, because the stringification process would yield `"(39)"` then instead of `"39"` in `printf`